### PR TITLE
sdk for stats API

### PR DESCRIPTION
**Pull request by dihan0604**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@dihan0604__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (dihan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by stankovski** [comment]
Please fix the tests and add coverage for the new operations.

**Comment by dihan0604** [comment]
Hi Denis, 
I have added recorded test as your suggested. Could you take a look and merge it? 

Thanks
Ding

File: src/ServiceManagement/Network/NetworkManagement/Microsoft.WindowsAzure.Management.Network.nuget.proj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     Microsoft.WindowsAzure.Management.Network
     -->
     <SdkNuGetPackage Include="Microsoft.WindowsAzure.Management.Network">
-      <PackageVersion>7.0.4</PackageVersion>
+      <PackageVersion>7.0.5</PackageVersion>
```
Comment (jianghaolu) [inline src/ServiceManagement/Network/NetworkManagement/Microsoft.WindowsAzure.Management.Network.nuget.proj]: Why is this version changed? Did you make any changes to the Network libraries?

**Comment by dihan0604** [comment]
We added some new API in Network libraries.

**Comment by jianghaolu** [comment]
Hi @dihan0604 I don't see any network library changes here, is there anything I'm missing?

**Comment by nupurkot** [comment]
LGTM

**Comment by dihan0604** [comment]
Have addressed the comment, please merge it.  I updated the express route version and revert the one for network

**Comment by jianghaolu** [comment]
@azuresdkci retest this please

**Comment by jianghaolu** [comment]
The master branch is failing due to another merged bad pull request. Should be fixed in a bit.

**Comment by yugangw-msft** [comment]
@dihan0604, please fetch latest master and merge, so to trigger CI run. We have reverted problematic PRs to clean up the current master branch

**Pull request merged**